.github/workflows/ci.yml

```
@@ -71,6 +71,8 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 18
+      - name: Install build-essentials
+        run: apt-get update && apt-get install build-essential -y
       - name: Install dependencies
         run: npm install
```

why not as part of "Install dependencies"?
```suggestion
      - name: Install dependencies
        run: |
        apt-get update && apt-get install build-essential -y
        npm install
```